File: Angular/SearchByTag/src/app/main/tag/tag.ts
export class Tag {
    private id: number;
    get Id(){
        return this.id;
    }
    set Id(value: number){
        this.id = value;
    }
    
    private item: string;
    get Item(){
        return this.item;
    }
    set Item(value: string){
        this.item = value;
    }

    private normalized: string;
    get Normalized(){
        return this.normalized;
    }
    set Normalized(value: string){
        this.normalized = value;
    }

    private idMain: number;
    get IdMain(){
        return this.idMain;
    }
    set IdMain(value: number){
        this.idMain = value;
    }
}

File: API/SearchByTag/SearchByTag/Core/Model/Tag.cs
using System;

namespace SearchByTag.Core.Model
{
    public class Tag
    {
        public long Id { get; set; }
        public String Item { get; set; }
        public String Normalized { get; set; }
        public long IdMain { get; set; }
        public Main Main { get; set; }
    }
}